Games/Retro_ML.SuperMario64/SM64Plugin.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Retro_ML.Configuration;
+using Retro_ML.Emulator;
+using Retro_ML.Game;
+using Retro_ML.Neural;
+using Retro_ML.Neural.Play;
+using Retro_ML.Neural.Train;
+using Retro_ML.Plugin;
+using Retro_ML.SuperMario64.Configuration;
+using Retro_ML.SuperMario64.Game;
+using Retro_ML.SuperMario64.Neural;
+using Retro_ML.SuperMario64.Neural.Train;
+using SharpNeat.BlackBox;
+
+namespace Retro_ML.SuperMario64;
+
+internal class SM64Plugin : IGamePlugin
+{
+    public string PluginGameName => "Super Mario 64";
+    public string PluginROMHeaderName => "SUPER MARIO 64     ";
```

Comment:
what are the spaces for in the end of the PluginROMHeaderName?

Reply:
In the ROM header, the game name always has a fixed length, and the rest of the characters are spaces